_articles/getting-started/import-from-1Password.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: article
+title: Import Your Account Data From 1Password
+category: Getting Started
+featured: true
+popular: true
+tags: [import, 1Password]
+---
+
+Importing your data from 1Password into bitwarden is easy.
+
+**From the 1Password application**
+
+1. Open 1Password on your computer. Enter your master password to unlock your vault.
+2. Select the vault you want to export. It’s not possible to export from “All Vaults”, so you’ll need to switch to a specific vault.
+3. Look under Categories for Logins. Click on it and highlight all the login items. On macOS, use command + a. On Windows, use control + a.
```

Comment:
Is this step necessary? The import tool should already filter out anything except web logins. The user should be able to just export all.